R/RLearner_classif_liquidSVM.R

```diff
@@ -0,0 +1,49 @@
+#' @export
+makeRLearner.classif.liquidSVM = function() {
+  makeRLearnerClassif(
+    cl = "classif.liquidSVM",
+    package = "liquidSVM",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "d", lower = 0L),
+      makeLogicalLearnerParam(id = "scale"),
+      makeIntegerLearnerParam(id = "threads", lower = -1L),
+      makeIntegerLearnerParam(id = "partition_choice", lower = 0L),
+      makeIntegerLearnerParam(id = "grid_choice", lower = -2L, upper = 2L),
+      makeIntegerLearnerParam(id = "adaptivity_control", lower = 0L, upper = 2L),
+      makeIntegerLearnerParam(id = "random_seed", default = 1),
```

Comment:
I didn't find any documentation on what the default random seed is -- where did you find this? It also shouldn't be tunable.